src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Tests/TestHelper.cs

```diff
@@ -17,34 +17,42 @@ public static List<Test> GetTestList() => new List<Test>()
             {
                 Id = 1,
                 Number = 1,
-                StartDate = DateTime.Today.AddDays(-1).AddHours(12),
-                EndDate = DateTime.Today.AddDays(-1).AddHours(23).AddMinutes(59),
+                StartDate = new DateTime(2020, 12, 1, 20, 0, 0),
```

Comment:
why have you changed that?

Reply:
Isn't it better to set up specific test start date? What if you'd run tests for example, in July? In other places in the code there are sections that subtract the specific dates in December from those dates.